src/FilenameParsing/HashFileIDHelper.php

```diff
@@ -124,9 +84,4 @@ private function truncate($hash)
     {
         return substr($hash ?? '', 0, self::HASH_TRUNCATE_LENGTH);
     }
-
-    public function lookForVariantRecursive(): bool
```

Comment:
Moved this method to the abstract class as it's identical to the implementation in `NaturalFileIDHelper`